src/e2e-test/regression-test/util/utils.js

```diff
@@ -0,0 +1,78 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+// TODO: consider using `util` nodule inside node env
+const _ = require('lodash');
+const handlers = require('./handlers');
+const MAX_COMPARISION_DEPTH = 10;
+
+const getGroundTruthFileName = testCase => {
+    if (!!testCase && !!testCase.templateFile && !!testCase.dataFile) {
```

Comment:
&& => ||?

Reply:
The condition statement is to ensure that all of the three variables are exist, so the "&&" is appropriate here. I think maybe the "!!" is a little confusing, in fact the statement `if (blahblah)` is exactly the same with `if (!!blahblah)`, the latter one is a trick for JS, but it's not recommended anymore officially. I'll remove the "!!" for better readability.